docs/get_started.ipynb

```diff
@@ -291,14 +287,37 @@
         "Additional TensorBoard plugins are automatically enabled when you log other types of data. For example, the Keras TensorBoard callback lets you log images and embeddings as well. You can see what other plugins are available in TensorBoard by clicking on the \"inactive\" dropdown towards the top right."
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "vsowjhkBdkbK"
+      },
+      "source": [
+        "## Hosting your TensorBoard to share with others\n",
+        "\n",
+        "[TensorBoard.dev](https://tensorboard.dev) is a free, public [TensorBoard](https://tensorflow.org/tensorboard) service that enables you to upload and share your ML experiments with everyone.  It provides a more permanent, shareable solution that doesn't require re-executing your notebook.\n",
```

Comment:
I suggest changing "upload and share your ML experiments with everyone" to something like "upload your TensorBoard logs and get a permalink that can be shared with everyone in academic papers, blog posts, social media, etc. This can enable better reproducibility and collaboration".